CONTRIBUTING.md

```diff
@@ -1,15 +1,34 @@
 # Contributing to Rust
 
-Thank you for your interest in contributing to Rust!
+Thank you for your interest in contributing to Rust! There are many ways to contribute
+and we appreciate all of them.
 
 To get started, read the [Contributing to Rust] chapter of the [rustc-dev-guide].
 
+## About the [rustc-dev-guide]
+
+The [rustc-dev-guide] is meant to help document how rustc –the Rust compiler– works,
+as well as to help new contributors get involved in rustc development. It is recommend
+to read and understand the [rustc-dev-guide] before making a contribution. This guide
+talks about the different bots in the Rust ecosystem, the Rust development tools,
+bootstrapping, the compiler architecture, source code representation, and more.
+
+## Getting help
+
+There are many ways you can get help when your stuck. Rust has many platforms for this:
```

Comment:
I would link to https://rustc-dev-guide.rust-lang.org/getting-started.html#asking-questions instead of duplicating the content here.

```suggestion
There are many ways you can get help when you're stuck. Rust has many platforms for this:
```

Reply:
You marked this as resolved, but you still inlined the info instead of using a link.

Reply:
Should I just leave the link there and not do anything else?

Reply:
Are the current changes what you are looking for? Sorry if I misunderstood

Reply:
No problem. These changes aren't exactly what I meant but they look good :)